licenses.yaml

```diff
@@ -136,6 +136,16 @@ source_paths:
 
 ---
 
+name: code adapted from Apache Knox KnoxSessionStore and ConfigurableEncryptor
+license_category: source
+module: extensions-core/druid-kubernetes-extensions
```

Comment:
Oh I mean, should this be `extensions/druid-pac4j`?

Reply:
Oh Man, I seem to be doing too much multi-tasking   and my head doesn't seem to be in right place.

that said I reverted back to "extensions/" as   that is used everywhere else. Removing module attribute altogether is the right thing to do I guess.

also created https://github.com/apache/druid/issues/9556

Reply:
Thank you!